bundles/mapping/mapmodule/MapModuleClass.ol.js

```diff
@@ -624,16 +625,18 @@ export class MapModule extends AbstractMapModule {
             return false;
         }
 
+        if (zoom && top && bottom && left && right) {
+            const zoomOut = top === bottom && left === right;
+            this.zoomToExtent(zoom, zoomOut, zoomOut);
+            view.setCenter(lonlat);
+            return true;
```

Comment:
Did you check does this trigger the AfterMapMoveEvent or do we need to add the notifyMoveEnd() call here to be used with view.setCenter()

Reply:
The event is sent.